### ISSUE TITLE
TextureEffect and uvTransform not working

### ISSUE THREAD

**Issue by funwithtriangles** [vanruesc/postprocessing#182]
I'm trying to scale the texture of `TextureEffect` so that it maintains the correct aspect ratio. To enable texture transforms, I believe I should be setting `uvTransform` to 1, like this:

```javascript
const webcamEffect = new TextureEffect({
  texture: camTexture,
})

webcamEffect.uvTransform = 1
```

Doing this causes a shader compile error and I'm not sure why.

Here's a sandbox with the same issue replicated (uncomment line 57 to see the error)
https://codesandbox.io/s/loving-tesla-emoux

**Comment by vanruesc** [comment]
Hi, thanks for reporting this!

The `uvTransform` setting is supposed to be a boolean flag, but the outcome will be the same if `1` is used instead. The documentation is wrong in this regard.

The shader error is caused by an awkward combination of compressed shader code that ends up with an `#ifdef` at the end of a long line of code:

```sh
55: uniform vec2 resolution;uniform vec2 texelSize;uniform float cameraNear;uniform float cameraFar;uniform float aspect;uniform float time;varying vec2 vUv;#ifdef ASPECT_CORRECTION
```

This error went unnoticed in the [texture demo](https://vanruesc.github.io/postprocessing/public/demo/#texture) because here the `SMAAEffect` comes before the `TextureEffect`.

A fix is on the way.

**Comment by vanruesc** [comment]
Fixed in `postprocessing@6.13.2`.
